src/components/dropdown/interfaces.ts

```diff
@@ -9,7 +9,7 @@ export interface OptionRendererProps<T> {
 
 export type HTMLInputProps = Omit<
   React.HTMLProps<HTMLInputElement>,
-  'onChange' | 'value'
+  'onChange' | 'value' | 'size' | 'defaultValue'
```

Comment:
If there are blocking issues, then I suggest you add this line also in the 2.11.2-branch. (To include it in 2.11.2 TK release)

Reply:
`size` and `defaultValue` where not part of the Dropdown interface at branch 2.11.2

Reply:
`defaultValue` is here since (almost?) the beginning of the component
but I agree that `size` was introduced by the feature branch SDS-INPUT-GROUP (=not released yet)

Reply:
I just checked, `defaultValue` is needed but there were no examples on the storybook :/

Reply:
There is one here https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-components/blob/master/stories/Dropdown.stories.tsx#L226